src/helpers/db.js

```diff
@@ -0,0 +1,136 @@
+import sqlite3 from 'sqlite3';
+
+const {
+    DB_FILE = 'data/database.db',
+} = process.env;
+
+// Open db
+let db = new sqlite3.Database(DB_FILE);
+
+/**
+ * Function to create required DBs if they do not exist
+ */
+export const createDBs = async () =>
+{
+    await db.exec("PRAGMA foreign_keys=ON");
+    await db.exec(`
+        CREATE TABLE IF NOT EXISTS jobs (
+        id TEXT,
+        name TEXT PRIMARY KEY,
+        request_id INTEGER DEFAULT 0,
+        last_reported_round INTEGER DEFAULT 0,
+        last_request_sent REAL DEFAULT 0,
+        last_submission_time REAL DEFAULT 0,
+        last_result REAL DEFAULT -1,
+        last_received_request_id INTEGER DEFAULT 0
+        );
+
+        CREATE TABLE IF NOT EXISTS rounds (
+        feed TEXT,
+        round_id INTEGER DEFAULT 0,
+        started_at REAL DEFAULT 0,
+        started_by TEXT,
+        submission_made INTEGER DEFAULT 0,
+        FOREIGN KEY (feed) REFERENCES jobs(name) ON DELETE CASCADE
+        );
+  `);
+}
```

Comment:
Issue with doing a DB creation like this, if there was a code update to update the tables the node operators need to manually delete the database to apply the changes. I suggest using migrations

Reply:
Fixed in https://github.com/jacquesvcritien/agoric-cl-middleware/commit/c24989fcb085b9832ee0afa48c85fbf2f708198e